lib/simplecov/configuration.rb

```diff
@@ -7,7 +7,7 @@
 # SimpleCov documentation for further info.
 #
 module SimpleCov
-  module Configuration
+  module Configuration # rubocop:disable all
```

Comment:
or wait is it just for this line? Maybe deserves explanatory comment .. don't want this being copied everywhere.